src/ExtendedXmlSerializer/IExtendedXmlSerializerConfig.cs

```diff
@@ -27,12 +27,14 @@ namespace ExtendedXmlSerialization
 {
     public interface IExtendedXmlSerializerConfig
     {
-        Type Type { get; }
-        int Version { get; set; }
+        Type Type { get; } // Consider making obsolete as logic is now contained within implementation.
```

Comment:
You right. This property can be removed but there is small work to do. If you would like you can do it.